LBHackney-IT/lbh-social-care
# added modal component

**What**  
- Added modal componet
- Added missing test dependency `identity-obj-proxy`
- Refactored `DeleteIcon`, renamed `CircleTimeIcon` following font awesome naming convention

<img width="1456" alt="Screenshot 2021-01-07 at 21 50 34" src="https://user-images.githubusercontent.com/435399/103950134-a8fd6b00-513c-11eb-8d66-2882b8bbf127.png">

**Note**
Probably we can polish the UI, but it's good enough to start using it

**How to test it**
- spin up storybook
- go to [http://localhost:6006/?path=/story/modal--default](http://localhost:6006/?path=/story/modal--default)
- check that is working fine
